src/environments/environment.ts

```diff
@@ -15,7 +15,16 @@ export const environment = {
   version: env.npm_package_version + '-dev',
   serverUrl: '/api',
   defaultLanguage: 'en-US',
-  supportedLanguages: ['en-US', 'fr-FR']
+  supportedLanguages: ['en-US', 'fr-FR'],
+  firebaseConfig: {
+    apiKey: 'AIzaSyAC9POPNO4FDdyUuCtly2NqJkNXu8zEbeA',
```

Comment:
Warto by to repo zrobic jako prywatne, chociaz boty i tak juz pewnie pobraly ten connection string ;p

Reply:
Wygenerowalem nowy - podajcie maile gmaila to dodam wam jako wspoltworcow do firebase i sobie skopiujecie

Reply:
warto ale to kosztuje na githubie - skladamy się?;)

Reply:
1  repo prywatne jest darmowe z tego co wiem